diff --git a/memflow-ffi/bindgen.sh b/memflow-ffi/bindgen.sh
index 13e84a9..e61bcc2 100755
--- a/memflow-ffi/bindgen.sh
+++ b/memflow-ffi/bindgen.sh
@@ -11,5 +11,5 @@ run_twice() {
 	fi
 }
 
-run_twice rustup run nightly cglue-bindgen +nightly -c cglue.toml -- --config cbindgen.toml --crate memflow-ffi --output memflow.h -l C
-run_twice rustup run nightly cglue-bindgen +nightly -c cglue.toml -- --config cbindgen.toml --crate memflow-ffi --output memflow.hpp -l C++
+run_twice cglue-bindgen -c cglue.toml -- --config cbindgen.toml --crate memflow-ffi --output memflow.h -l C
+run_twice cglue-bindgen -c cglue.toml -- --config cbindgen.toml --crate memflow-ffi --output memflow.hpp -l C++
